Source/JavaScriptCore/API/JSContext.mm

```diff
@@ -162,7 +162,7 @@ - (JSValue *)dependencyIdentifiersForModuleJSScript:(JSScript *)script
     }
 
     auto scope = DECLARE_CATCH_SCOPE(vm);
-    JSC::JSArray* result = globalObject->moduleLoader()->dependencyKeysIfEvaluated(globalObject, JSC::jsString(vm, String([[script sourceURL] absoluteString])));
+    JSC::JSArray *result = globalObject->moduleLoader()->dependencyKeysIfEvaluated(globalObject, JSC::jsString(vm, String([script sourceURL].absoluteString)));
```

Comment:
I think the style is to use WebKits C++ * placement for C++ types and Obj-C style for Obj-C classes.